test-container/src/main/resources/kafka-versions.txt

```diff
@@ -0,0 +1,3 @@
+2.4.0
```

Comment:
why do we need this file?

Reply:
This is the file generated in the build phase of the whole project. So by that, we are avoiding problem when we use this dependency in the different project. For instance, if I use it inside Bridge. 
----
Previously, I had implementation with parsing the `kafka-versions.yaml` in Runtime by this is not possible because that file `kafka-versions.yaml` will not be inside another repository. 
----